File: AIPP course/Assignment 18/Student_names_18(4).py
# Function to print student names
def print_students(students):
    print("Student List:")
    for name in students:
        print(name)

# User input
user_input = input("Enter student names separated by commas: ")

# Convert input string to list
students = [name.strip() for name in user_input.split(",")]

# Test the function
print_students(students)

File: AIPP course/Assignment 18/student_names_18(4).js
// CodeChef-compatible JavaScript code

function printStudents(students) {
    console.log("Student List:");
    students.forEach(name => console.log(name));
}

const fs = require("fs");

// Read input from STDIN
let input = fs.readFileSync(0, "utf8").trim();

// Convert input into array
let students = input.split(",").map(s => s.trim());

// Call the function
printStudents(students);
